e2e/tests/migration_steps_test.go

```diff
@@ -20,68 +84,133 @@ const (
+	s.suite.Before()
+
+	// Initialize the morse & shannon key indices.
+	s.nextShannonKeyIdx()
+	s.nextMorseKeyIdx()
+
+	s.existingUnstakedBalanceUpokt = cosmostypes.NewInt64Coin(volatile.DenomuPOKT, 0)
+	if shannonDestAddr, isFound := s.getShannonKeyAddress(); isFound {
+		if _, isFound = s.queryAccount(shannonDestAddr); isFound {
+			upoktBalanceInt := s.getAccBalance(s.getShannonKeyName())
+			upoktBalanceCoin := cosmostypes.NewInt64Coin(volatile.DenomuPOKT, int64(upoktBalanceInt))
+			s.existingUnstakedBalanceUpokt = upoktBalanceCoin
+		}
+	}
+}
+
+// TestMigrationWithFixtureData runs the migration_fixture.feature file ONLY.
 // To run this test use:
-// 	$ make test_e2e_migration
-// OR
-// 	$ go test -v ./e2e/tests/migration_steps_test.go -tags=e2e,manual
-func TestMigrationFeatures(t *testing.T) {
-	gocuke.NewRunner(t, &migrationSuite{}).Path("migration_*.feature").Run()
+//
+// The @oneshot tag indicates that a given feature is non-idempotent with respect
+// to its impact on the network state. In such cases, a complete network reset
+// is required before running these features again.
+//
+//	$ make test_e2e_migration_fixture
+func TestMigrationWithFixtureData(t *testing.T) {
+	gocuke.NewRunner(t, &migrationSuite{}).
+		Path("migration_fixture.feature").
+		Run()
 }
 
 func (s *migrationSuite) ALocalMorseNodePersistedStateExists() {
-	s.Skip("TODO_UPNEXT(@bryanchriswhite, #1034): Implement.")
+	homeDir, err := os.UserHomeDir()
+	require.NoError(s, err)
+
+	// Check for the $HOME/.pocket/application.db, etc. files.
+	pocketDir := path.Join(homeDir, defaultMorseDataDir)
+	for _, dbFileName := range morseDatabaseFileNames {
+		dbPath := path.Join(pocketDir, dbFileName)
+		_, err = os.Stat(dbPath)
+		require.NoErrorf(s, err, "expected %s to exist", dbPath)
+	}
 }
 
 func (s *migrationSuite) NoMorseclaimableaccountsExist() {
-	s.Skip("TODO_UPNEXT(@bryanchriswhite, #1034): Implement.")
+	morseClaimableAccounts := s.queryListMorseClaimableAccounts()
+	require.Lessf(s, len(morseClaimableAccounts), 1, "expected 0 morse claimable accounts, got %d", len(morseClaimableAccounts))
 }
 
 func (s *migrationSuite) TheShannonDestinationAccountIsStakedAsAn(actorType actorTypeEnum) {
 	s.Skip("TODO_UPNEXT(@bryanchriswhite, #1034): Implement.")
 }
 
-func (s *migrationSuite) TheShannonStakeEqualsTheOfTheMorseclaimableaccount(actorType actorTypeEnum, totalTokensStakePct float64) {
-	s.Skip("TODO_UPNEXT(@bryanchriswhite, #1034): Implement.")
-}
-
 func (s *migrationSuite) TheShannonStakeIncreasedByTheOfTheMorseclaimableaccount(actorType actorTypeEnum, totalTokensStakePct float64) {
 	s.Skip("TODO_UPNEXT(@bryanchriswhite, #1034): Implement.")
 }
 
-func (s *migrationSuite) TheMorsePrivateKeyIsUsedToClaimAMorseclaimableaccountAsAnWithoutSpecifyingTheStakeAmount(actorType actorTypeEnum) {
+func (s *migrationSuite) TheMorsePrivateKeyIsUsedToClaimAMorseclaimableaccountAsAn(actorType actorTypeEnum) {
 	s.Skip("TODO_UPNEXT(@bryanchriswhite, #1034): Implement.")
```

Comment:
Please do a global grep for `TODO_UPNEXT(@bryanchriswhite` and add mainnet where necessary